src/Clustering/KMeans.php

```diff
@@ -43,8 +48,15 @@ public function cluster(array $samples): array
         $clusters = [];
         foreach ($space->cluster($this->clustersNumber, $this->initialization) as $cluster) {
             $clusters[] = $cluster->getPoints();
+            $arrayCluster = $cluster->toArray();
+            $this->centronoids[] = $arrayCluster['centroid'];
```

Comment:
maybe:
```suggestion
            $this->centronoids[] = $cluster->toArray()['centroid'];
```

or even better:
```suggestion
            $this->centronoids[] = $cluster->centroid();
```
and add `centroid` method to `Cluster`